monai/transforms/spatial/dictionary.py

```diff
@@ -743,6 +775,11 @@ def __init__(
         self.mode = ensure_tuple_rep(mode, len(self.keys))
```

Comment:
here `align_corners ` is not input into `Affine`

Reply:
please help revise if needed, it's an input argument in the constructor though https://github.com/Project-MONAI/MONAI/blob/f389d87a6b3f5a0a726c88e0c9fca95f2379f84b/monai/transforms/spatial/dictionary.py#L760-L769